src/index.html

```diff
@@ -17,15 +17,14 @@
 
   <script>
   var config = {
-    apiEndpoint: 'http://localhost:8000',
+    // apiEndpoint: 'http://localhost:8000',
+    apiEndpoint: "https://api.g8s.gauss.eu-central-1.aws.gigantic.io",
```

Comment:
You forgot this here 😬

Reply:
Fixed meanwhile in https://github.com/giantswarm/happa/pull/808/commits/3a930e188771973aa50dc7fcdfd3fc3965b6cfe8